src/Microsoft.DotNet.Build.Tasks.net45/project.json

```diff
@@ -8,7 +8,7 @@
     "NuGet.Versioning": "3.6.0-rc-1987",
     "System.Collections.Immutable": "1.1.37",
     "System.Reflection.Metadata": "1.1.0",
-    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0"
+    "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24806-04"
```

Comment:
The package closure isn't the real trouble necessarily but it might point to any new dependencies that the assembly might have. I don't know if there are any but that is what I'm worried about. If the assembly it is including now has all its dependencies satisfied already then I'm good with this change.